translations/translations.json

```diff
@@ -7076,7 +7076,8 @@
       "adjustFilters": "Filter locations",
       "onlyCurrentlyOpen": "only show currently open locations",
       "poiCategories": "Categories",
-      "showPois": "Show locations",
+      "showPois": "Show {{count}} places",
+      "showPois_one" : "Show {{count}} place",
```

Comment:
Could you adjust those similar to the german translations, i.e. adding `_zero`, `_one`, `_two`, `_few`, `_many` and `_other` translations.

Reply:
Alright, i'm on it.

Reply:
I am reading about test with typescript to understand exactly why the check in circleci fails.

Reply:
We can also do a call some time to help you if you want. At the moment it fails because of prettier (which formats our code, e.g. new lines, indentations, spaces...) in the translations.json file. Perhaps there is a trailing comma or similar.

Reply:
A call would be nice.